modified:   main.py

LutzBY opened the pull request:
Изменена выдача сообщения функции send_message в функции play_dice. Сообщение более не содержит ссылку на пользователя, только имя.
Изменено для того, чтобы бот не тегал список losers пользователей без необходимости и извещение/тег приходило только победителю play_dice

LutzBY committed:
modified:   main.py
изменено сообщение с кубиками (не имя-ссылка а просто имя) и cocial замено на social

dsankouski commented on main.py:
```diff
@@ -95,7 +95,7 @@ def play_dice(chat_id, users):
         user_name = user.first_name
         bot.send_message(chat_id=chat_id,
-                         text="[{}](tg://user?id={}):".format(user_name, user_id),
+                         text="{}:".format(user_name), # я с маркдаун не сталкивался ранее я хз надо ли тут оставлять []
```
не нужно. [] нужны для ссылки, так как нам ссылка не нужна, скобки тоже. Удали комментарий

LutzBY replied:
Сделано

LutzBY committed:
modified:   main.py
Убран комментарий в строке 98

dsankouski commented on main.py:
```diff
@@ -128,7 +128,7 @@ def send_assigned_user_msg(chat_id, user):
     user_id = user.id
     bot.send_message(chat_id=chat_id,
-                     text="[{}](tg://user?id={}), ".format(user_name, user_id) + _("I don't know how to organize cocial events, could you please, help me?"),
+                     text="[{}](tg://user?id={}), ".format(user_name, user_id) + _("I don't know how to organize social events, could you please, help me?"),
```
В gettext это сообщение используется как ключ для поиска соответствующего перевода. То есть необходимо еще поправить файлы в locale каталоге. Так же это изменение должно быть в отдельном коммите, чтобы не смешивать изменения в логике и переводах

dsankouski commented:
переведи сообщение коммита на английский, и добавь туда комментарий из пулл реквеста.

LutzBY committed:
modified:   main.py
Made changes to text content of send_message function inside play_dice function. Now instead of sending Name and link to account it sends only Name.
Reason: not to ping everyone in list 'losers', only notice and ping the winner of play_dice function
